bin/oref0-pump-loop.sh

```diff
@@ -588,7 +585,7 @@ function calculate_iob {
 
 function invoke_pumphistory_etc {
     check_clock 2>&3 >&4 || return 1
-    read_pumphistory 2>&3 >&4 || return 1
+    read_pumphistory 2>&3  || return 1
```

Comment:
I wanted to be able to show in the logs when a full refresh is done vs just a pumphistory update, but this change kinda messes up the log a little, not sure what's the right approach.